index.js

```diff
@@ -464,7 +478,7 @@ exports.isZeroAddress = function (address) {
  */
 exports.toChecksumAddress = function (address) {
   address = exports.stripHexPrefix(address).toLowerCase()
-  const hash = exports.sha3(address).toString('hex')
+  const hash = exports.keccak(address).toString('hex')
```

Comment:
Ok. Also checked that there is nothing forgotten in ``index.js`` by pre-scrolling the ``master`` branch version to the next position where I would have inserted/changed something.